src/containers/scenes/mobile/nrc-landing-scene-mobile/nrc-landing-scene-mobile-component.jsx

```diff
@@ -63,10 +64,36 @@ function NrcLandingMobileComponent({
 }) {
   const t = useT();
 
+  const isLandscape = useLandscape();
+
+  const [updatedSceneSettings, setUpdatedSceneSettings] =
+    useState(sceneSettings);
+
+  useEffect(() => {
+    if (isLandscape) {
+      setUpdatedSceneSettings({
+        ...sceneSettings,
+        padding: {
+          left: -450,
+          top: 50,
+        },
+      });
+    }
+    if (!isLandscape) {
+      setUpdatedSceneSettings({
+        ...sceneSettings,
+        padding: {
+          left: 0,
+          top: -220,
+        },
+      });
+    }
+  }, [isLandscape]);
```

Comment:
These settings are not updating when landscape mode is setting on dev tools, so im not sure is gonna work on the mobile